client/my-sites/backup/backup-contents-page/file-browser/file-info-card.tsx

```diff
@@ -80,8 +80,8 @@ function FileInfoCard( {
 
 	const handleDownloadError = useCallback( () => {
 		setIsProcessingDownload( false );
-		dispatch( onProcessingDownloadError() );
-	}, [ dispatch ] );
+		notices.showError( __( 'There was an error processing your download. Please, try again.' ) );
```

Comment:
Similar to the comment above:

```suggestion
		notices.showError( __( 'There was an error processing your download. Please try again.' ) );
```